Havit.Blazor.Components.Web.Bootstrap/Forms/SearchBox/HxSearchBox.razor.css

```diff
@@ -17,11 +17,6 @@
     right: .75rem;
 }
 
-::deep .hx-search-box-input-with-search-icon {
```

Comment:
@crdo this class is necessary when `InputGroupStartText` or `InputGroupStartTemplate` is set because otherwise the corners aren't rounded.